library/src/main/java/kwasm/format/text/Utils.kt

```diff
@@ -225,6 +225,18 @@ fun Int.canon(context: ParseContext? = null): Long =
 
 private fun Int.toStringAsCodepoint(): String = String(intArrayOf(this), 0, 1)
 
+fun getOperationAndParameters(sequence: CharSequence, context: ParseContext?): Pair<CharSequence, List<CharSequence>> {
+    if (sequence.first() != '(') {
+        throw ParseException("Expecting opening ( for Operation", context)
+    }
+    if (sequence.last() != ')') {
+        throw ParseException("Expecting closing ) for Operation", context)
+    }
+
+    val splitSequence = sequence.substring(1, sequence.lastIndex).split("\\s".toRegex())
+    return Pair(splitSequence[0], splitSequence.subList(1,splitSequence.lastIndex+1))
+}
+
```

Comment:
This is good for now, but we'll need to migrate over to using tokenized stuff once I land that. Hopefully won't take too long to migrate things.

Reply:
Sounds good